Phoenix/PHContext.h

```diff
@@ -10,6 +10,10 @@
 
 - (void) load;
 
+#pragma mark - Editing
+
+- (NSString *) primaryConfigurationPath;
```

Comment:
Expose this by adding a read-only @property to the interface instead of creating an explicit getter, see https://github.com/kasper/phoenix/blob/master/Phoenix/PHKeyHandler.h#L13 for an example.